vnext/Scripts/Microsoft.ReactNative.Managed.CodeGen.SignConfig.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
+  <job platform="" configuration="" dest="__INPATHROOT__" jobname="microsoftreactnative" approvers="">
+    <!-- Overwrite the input files to simplify downstream things like nuget package generation. -->
+
+    <!-- Is Authenticode right here? Do we need strong-signing for dotnet core binaries? -->
```

Comment:
Since this is a netcore3.1 tool and nobody needs to reference this programatically I'm totally fine not strong-naming it.

See [Strong Name Signing](https://github.com/dotnet/runtime/blob/4f9ae42d861fcb4be2fcd5d3d55d5f227d30e723/docs/project/strong-name-signing.md) doc on the netcore runtime page.